docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -8,19 +8,26 @@ weight: 8
 
 # Public dashboards
 
-> **Note:** This is an opt-in alpha feature.
+{{% admonition type="Note" %}}
 
-> **Caution:** Making your dashboard public could result in a large number of queries to the datasources used by your dashboard.
-> This can be mitigated by utilizing the enterprise [caching](https://grafana.com/docs/grafana/latest/enterprise/query-caching/) and/or rate limiting features.
+This feature is in [public preview](/docs/release-life-cycle/).
 
-Public dashboards allow you to share your Grafana dashboard with anyone. This is useful when you want to expose your
-dashboard to the world.
+{{% /admonition %}}
+
+{{% admonition type="Caution" %}}
+
+Making your dashboard public could result in a large number of queries to the data sources used by your dashboard.
+This can be mitigated by utilizing the enterprise [caching]({{< relref "../../administration/data-source-management/#query-caching" >}}) and/or rate limiting features.
+
+{{% /admonition %}}
+
+Public dashboards allow you to share your Grafana dashboard with anyone. This is useful when you want to make your dashboard available to the world without requiring access to your Grafana organization. This differs from [dashboard sharing]({{< relref "../share-dashboards-panels" >}}), which either requires recipients to be part of the same Grafana organization or provides limited information, as with a snapshot.
```

Comment:
I'm not sure about the word `recipients` here because it sounds like anyone could be the recipient of a dashboard but they must be Grafana users. Please disregard my message if we are using this word in other parts of the docs.

Reply:
Hmm, that's a good point. I do mean recipient, because a recipient could be a Grafana user or not. But if that's making this confusing, then maybe it's not the best way to put this. What about this:

```suggestion
Public dashboards allow you to share your Grafana dashboard with anyone. This is useful when you want to make your dashboard available to the world without requiring access to your Grafana organization. This differs from [dashboard sharing]({{< relref "../share-dashboards-panels" >}}), which either requires recipients to be users in the same Grafana organization or provides limited information, as with a snapshot.
```

Reply:
Great!